doc/classes/File.xml

```diff
@@ -327,6 +327,17 @@
 				Stores the given array of bytes in the file.
 			</description>
 		</method>
+		<method name="store_csv_line">
+			<return type="void">
+			</return>
+			<argument index="0" name="values" type="PoolStringArray" default="&quot;,&quot;">
+			</argument>
+			<argument index="1" name="delim" type="String" default="&quot;,&quot;">
+			</argument>
+			<description>
+				Store the given [PoolByteArray] in the file as a line formated in the CSV (Comma Separated Values) format. You can pass a different delimiter to use other than the default "," (comma).
```

Comment:
`PoolByteArray` -> `PoolStringArray`
formated -> formatted

I'd also add that the delimiter must be 1 char long.